app/src/bf-game/index.js

```diff
@@ -174,10 +180,32 @@ export function drawCard(gameState) {
  * @param {CardType} card The card the player chose to play
  * @returns {GameState} New state of the game.
  */
-export function playCard(gameState, card) {
+export function playCard(gameState, userId, card) {
   const newGameState = deepClone(gameState);
+  let discard = true;
 
-  if (
+  // remove card from player's hand
+  // if ResolvingLoot, currentPlayer is not the one who played the card...
+  const playerHand = newGameState.hands[userId];
```

Comment:
I dont think we need to add a `userId` param to the `playCard` function. We can check the phase and use a different method to get the current player id depending on it.

Or for now, we can just check for `lootTargetId` and fallback to good ol' `getCurrentPlayerId()` ¯\\\_(ツ)\_/¯

```suggestion
  // if ResolvingLoot, currentPlayer is not the one who played the card...
  const playerHand = newGameState.hands[gameState.lootTargetId || getCurrentPlayerId()];
```

Reply:
We are going to need this anyway for the Deny card I think.